app/controllers/api/cocktails_controller.rb

```diff
@@ -0,0 +1,18 @@
+class Api::CocktailsController < ApplicationController
+  def index
+    response = RestClient.get(
+      "https://www.thecocktaildb.com/api/json/v2/9973533/filter.php",
+      {
+        params: {
+          i: params[:q],
+        },
```

Comment:
are these colons needed? not a big thing, but it looks like we're expecting multiple of these?

Reply:
Done